app/assets/javascripts/sessions_new.js.coffee

```diff
@@ -0,0 +1,7 @@
+$ ->
```

Comment:
Even though we have coffeescript in our repo, we're trying to only bring in new JS as plain old JS or ES6.  Do you mind converting this?

Reply:
no problem. sorry you had to tell me the same thing twice - do to the various rebases we lost a few of the things we fixed from the last PR. I'm going to go back and make sure we got all of those.

Reply:
No worries.  We haven't done anything to make it apparent without having to rely on memory.